official-antistasi-community/A3-Antistasi
# Garage limit garaging and state preservation at hq

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    as per dev meeting 28.Aug 2021:
    - Garaging has been limited to locations with flags present, these are: Outposts, Seaports, Airports/Bases, Resources, and Factories
    - Vehicles left near HQ will maintain their state on save/load.

### Please specify which Issue this PR Resolves.
closes #2091 #2029 #605 

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
1) get a vehicle, damage it, save with vehicle at HQ, reload. vehicle should be restored with the state preserved.
2) try to garage anywhere but at one of the aforementioned locations.

********************************************************
Notes:
